### ISSUE TITLE
Agents: can't figure out how to tab to the account menu

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#309613]
Testing #309100

<img width="678" height="186" alt="Image" src="https://github.com/user-attachments/assets/ce6baba2-dc09-4be8-b518-5e70511d3eab"/>

Try to tab to the account menu all the way on the right.

**Comment by meganrogge** [comment]
This was working for me when the `Update` button was present to the left of that. But doesn't work otherwise.
